### PR TITLE
assembly Language complete Resources.

### LINKED ISSUE #923: 💡 [FEATURE] - Add Assembly language complete resources.
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

_No response_

### Explain your solution

I want to add assemble lang programming section complete resources..

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by jayraj175coder**
# Pull Request
I add assembly language resources..

## Description
Please include a summary of the changes and the related issue.  

Closes #923 

## Type of change
- [x] Adding a new Resource
- [ ] Adding a new Category
- [ ] Documentation enhancement or fixes
- [ ] Website enhancement or fixes

## Checklist:
- [ ] I read carefully [CONTRIBUTING.md](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Screenshots (if appropriate):

image

### CODE REVIEW

File: Programming Languages/Assembly/readme.md [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+>> Characteristics
+
+Low-Level Language: Assembly language is considered low-level because it is closely related to the hardware's machine code instructions.
+
+Hardware Specific: Each type of processor has its own assembly language, meaning code written for one type of CPU won't run on another without modification.
+
+Mnemonic Codes: Instructions are written using mnemonic codes, which are abbreviations of the operation (e.g., MOV for move, ADD for addition).
+
+Registers and Memory: Programmers have direct control over the CPU's registers and memory addresses.
+Efficiency: Code written in assembly language can be highly optimized for performance and efficiency.
```
Comment (jfmartinz) [inline Programming Languages/Assembly/readme.md:13]:
```suggestion

```

File: Programming Languages/Assembly/readme.md [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+    <td>A chapter from the Pro Git book that covers the basics of branching and merging in Git.</td>
+  </tr>
+</table>
+
+## Youtube
+> The "YouTube" section features a compilation of YouTube channels and playlists dedicated to Assembly. These resources offer a wealth of video content, including tutorials, walkthroughs, live coding sessions, and discussions, making them valuable learning resources for developers interested in Assembly.
+### YouTube Channels
+<table width="100%">
+ <tr>
+   <th>Channel Name</th>
```
Comment (jfmartinz) [inline Programming Languages/Assembly/readme.md:173]:
```suggestion
   <th>Resource Name</th>
```

File: Programming Languages/Assembly/readme.md [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+ <tr>
+   <td><a href="https://www.youtube.com/playlist?list=PL3SAXYUEnrabDbKeOiJnLDRRWzIFg9pIn">Zeeshan Academy</a></td>
+   <td>Zeeshan Academy  offers in-depth Assembly tutorials and courses taught by industry experts, covering various aspects of the development.</td>
+ </tr>
+</table>
+
+### YouTube Playlists for Learning Assembly
+<table width="100%">
+ <tr>
+   <th>Playlist Name</th>
```
Comment (jfmartinz) [inline Programming Languages/Assembly/readme.md:193]:
```suggestion
   <th>Resource Name</th>
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by jayraj175coder** [comment]
Done.

**Review by jfmartinz — APPROVED** [review]
LGTM

**Pull request merged**